ontotext-yasgui-web-component/src/services/utils/visualisation-utils.ts

```diff
@@ -0,0 +1,73 @@
+import {Orientation, RenderingMode} from '../../models/yasgui-configuration';
+import {HtmlElementsUtil} from './html-elements-util';
+
+export class VisualisationUtils {
+
+  static setRenderMode(hostElement: HTMLElement, newMode: RenderingMode): void {
+    const modes: string[] = Object.values(RenderingMode);
+    const yasgui = HtmlElementsUtil.getOntotextYasgui(hostElement);
+    yasgui.classList.remove(...modes);
+    yasgui.classList.add(newMode);
+  }
+
+  static changeRenderMode(hostElement: HTMLElement, neMode: RenderingMode): void {
+    VisualisationUtils.unselectAllHeaderButtons(hostElement);
+    let button;
+    switch (neMode) {
+      case RenderingMode.YASQE:
+        button = HtmlElementsUtil.getYasqeButton(hostElement);
+        VisualisationUtils.setRenderMode(hostElement, RenderingMode.YASQE);
```

Comment:
Why do you do this? I don't see why the switch is needed at all at the moment

Reply:
Refactored